tests/Governance/GovernanceBravo/timelock.ts

```diff
@@ -0,0 +1,81 @@
+import chai from "chai";
+import { ethers, network } from "hardhat";
+
+const { expect } = chai;
+
+describe("TimelockV8 Tests", () => {
+  let deployer;
+  let timelock;
+  let testTimelock;
+
+  before(async () => {
+    [deployer] = await ethers.getSigners();
+    const timelockFactory = await ethers.getContractFactory("TimelockV8");
+    timelock = await timelockFactory.deploy(deployer.address, 4000);
+
+    const testTimelockFactory = await ethers.getContractFactory("TestTimelockV8");
+    testTimelock = await testTimelockFactory.deploy(deployer.address, 10);
+  });
+
+  it("Production timelock returns constant values", async () => {
+    expect(await timelock.getGracePeriod()).to.equal("1209600");
+    expect(await timelock.getMinimumDelay()).to.equal("3600");
+    expect(await timelock.getMaximumDelay()).to.equal("2592000");
+  });
+
+  it("Production timelock requires setting appropriate delay", async () => {
+    await deployer.sendTransaction({
+      to: timelock.address,
+      value: ethers.utils.parseEther("1.0"),
+    });
+
+    await network.provider.request({
+      method: "hardhat_impersonateAccount",
+      params: [timelock.address],
+    });
```

Comment:
We could do here something similar to [`initMainnetUser`](https://github.com/VenusProtocol/isolated-pools/blob/develop/tests/hardhat/Fork/utils.ts#L28)